README.md

```diff
@@ -147,6 +147,7 @@ for other operating systems. These packages are not guaranteed to be up-to-date.
         * `zypper install micro-editor` (SUSE)
         * `eopkg install micro` (Solus).
         * `pacstall -I micro` (Pacstall).
+        * `su - -c "apt-get install micro"` (ALT)
```

Comment:
But isn't this the same as with `Debian` or `Ubuntu`, where you just have to use `apt(-get)` with sufficient permission to install a package from the package repository? Especially since `ALT Linux` uses a mixture of `APT-RPM`.

Reply:
This is not quite true. ALT Linux does not support apt, only apt-get. Otherwise, you are generally right, but I think a separate line with the installation will not harm anyone, and some will even make it better. In addition, it does not have sudo support by default, so it would be nice to show how to get around this.

Reply:
That's the reason why I wrote `apt(-get)` and not just `apt`. :wink: 
We didn't wrote about the usage of `sudo` or `su` for the others too, since it's common sense, that you need proper permissions for the installation.

Reply:
I agree, but nevertheless, what is the problem with specifying the distribution separately? It's obviously not going to do anything wrong. Or is there still a reason not to take PR?

Reply:
Also, this distribution is little known in the community, so why not help it become a little more popular?

Reply:
Sorry, but what about my PR? Will you accept it or are there any problems?

Reply:
I'm more for the form of...
```
* `apt-get install micro` (for `apt-get`-only based Distributions)
``` 
...which is more generic and does fit the scenario as well. Furthermore I'm against highlighting the privilege escalation, since this then would be necessary for all other cases too. There is no need to mentioned this for one distribution only.

Reply:
> I'm more for the form of...
> 
> ```
> * `apt-get install micro` (for `apt-get`-only based Distributions)
> ```
> 
> ...which is more generic and does fit the scenario as well.

That would be misleading because using `apt` as the package manager does not guarantee that the distro has packaged micro in the default repositories. Mentioning distro by name is also good because it can be searched with Ctrl-f, and beginners may not even understand their distro is "apt-get based".

Reply:
So we agree to ```* `apt-get install micro` (ALT)```?

Reply:
Why not simply `` * `apt-get install micro` (Ubuntu, Debian, ALT) ``?
`apt-get` works wherever `apt` works, right?

Reply:
> So we agree to `` * `apt-get install micro` (ALT) ``?

Yes

Reply:
> Why not simply `` * `apt-get install micro` (Ubuntu, Debian, ALT) ``? `apt-get` works wherever `apt` works, right?

This is not a very good way, since Ubuntu and Debian usually use only apt, and apt-get is usually not used, as it is considered outdated

Reply:
> Why not simply `` * `apt-get install micro` (Ubuntu, Debian, ALT) ``? `apt-get` works wherever `apt` works, right?

Speaking for myself I prefer `apt` over `apt-get`, since it's a shorter front-end command to `dpkt` while providing some `apt-cache` stuff too:
https://www.howtogeek.com/791055/apt-vs-apt-get-whats-the-difference-on-linux/

It's more a matter of personal preference.
For me it feels like adding every single distro adding this package to this README. Do we plan to do this for all flavors and such?

Reply:
I'm sorry, do I understand correctly that we decided to stop at the option below?

```* `apt-get install micro` (ALT)?```

If so, is there anything else I need to do?

Reply:
> @ All Let me guess, but `` * `apt(-get) install micro` (Ubuntu, Debian, ALT) `` is too complicated to understand for beginners, right?

I think yes. It's better to separate them

Reply:
In general, you can do this, but the ALT must still be specified. But for the sake of truth, it seems to me that some will not understand why an apt-get is suddenly offered for the same Ubuntu

Reply:
> @ All Let me guess, but `` * `apt(-get) install micro` (Ubuntu, Debian, ALT) `` is too complicated to understand for beginners, right?

I'd rather keep the command copy-pasteable. I don't have strong preference between these two, either is fine IMO:
```md
* `apt-get install micro` (Ubuntu, Debian, ALT Linux)
```
or 
```md
* `apt install micro` (Ubuntu, Debian)
* `apt-get install micro` (ALT Linux)
```

Reply:
Personally, I prefer the second option with separate commands.

Reply:
I'm fine either.

Reply:
> ```
> * `apt install micro` (Ubuntu, Debian)
> * `apt-get install micro` (ALT Linux)
> ```

Then we take this one and should be fine. :)

Reply:
> > ```
> > * `apt install micro` (Ubuntu, Debian)
> > * `apt-get install micro` (ALT Linux)
> > ```
> 
> Then we take this one and should be fine. :)

Great! I agree

Reply:
Done!

Reply:
I'm waiting

Reply:
thanks